packages/firestore/test/util/equality_matcher.ts

```diff
@@ -71,8 +71,8 @@ export function addEqualityMatcher() {
 
           this.assert(
             customDeepEqual(left, right),
-            'expected #{this} to roughly deeply equal #{exp}',
-            'expected #{this} to not roughly deeply equal #{exp}',
+            'expected #{act} to roughly deeply equal #{exp}',
```

Comment:
What are `act` and `exp` here? Could you add a comment if this is actually correct?

Reply:
Without this change, these assertions provide little value. `this` is the object that is asserted on, and not the asserted value. See how it's done here: https://github.com/chaijs/chai/blob/3ace4a0bbd14cb3fad45067c29fed9880936e3b7/lib/chai/core/assertions.js#L1170